docs/source/excel_submission.rst

```diff
@@ -0,0 +1,362 @@
+.. role:: raw-html-m2r(raw)
+   :format: html
+
+
+================
+Excel Submission
+================
+
+
+Overview
+--------
+
+
+* Metadata and data can be submitted to our platform using Microsoft Excel WorkBooks that describe related items in separate sheets.
+* This section provides detailed information on how to use the WorkBooks.
+* A link to a detailed example of a filled out spreadsheet is forthcoming.
+* Based on the type of experiment(s) for which you plan to submit data, the data wranglers can provide you with an Excel WorkBook containing several WorkSheets.
+* Each sheet corresponds to an Item type in our metadata database.
+* The workbook provided should contain all the sheets that you may need for your submission.
+* Each sheet should also contain all the data fields that can be submitted for that Item type.
+* Depending on if you have submitted data before or if you are using shared reagents that have been submitted by other labs, you may not need to provide information on every sheet or in every field.
+
+**Organization of the Workbook**
+
+
+* Generally, it makes sense to begin with the left most sheet in the workbook as the sheets in a workbook are ordered so that Items that have fields that take a reference to another Item as their value appear ‘after’ i.e. to the right of that Item’s sheet in the workbook.
+* A sheet for an Item starts with a row of field names.
+* *Absolutely required fields are marked with a leading asterisk (eg. \*\ experiment_type).* - failure to supply a value in these fields will cause an error
+* The second row of the sheet indicates the type of the information expected for the fields.
+* The third row includes a description of each of the fields.
+* In some cases the values that you can submit for a particular field are constrained to a specific set of terms and when this is the case the possible values are shown in the fourth row.
+* Any row that starts with “#” in the first column will be ignored, so you can add non-data rows for your own use.
+* However, **PLEASE NOTE THAT THE FIRST 2 ROWS OF A SHEET SHOULD NOT BE MODIFIED.**
```

Comment:
I'm not sure how valid all this information is for us vs. 4DN here. Consider excluding until such things are settled.